docs/Technical Reference.md

```diff
@@ -318,3 +371,49 @@ Note the comma after the object. Your trust relationship should now look somethi
 ```
 
 You can now configure a Quilt Role with this role (using the Catalog's admin panel, or `quilt3.admin.create_role`).
+
+## Backup and Recovery
+
+All data and metadata in Quilt is stored in S3. S3 data is automatically backed up (replicated across multiple available zones). To protect against accidental deletion or overwriting of data, we strongly recommend enabling object versioning for all S3 buckets connected to Quilt.
+
+No data will be lost if a Quilt stack goes down. The Quilt search indexes will be automatically rebuilt when buckets are added to a new stack.
+
+### Region Failure
+To protect against data loss in the event of a region failure, enable
+[S3 Bucket Replication](https://aws.amazon.com/s3/features/replication/) on all S3 buckets.
+
+## Emergency Maintainance
+See [Troubleshooting](Troubleshooting.md)
+
+## Support
+Support is available to all Quilt customers by:
+* online chat (in the Quilt catalog)
+* email to [support@quiltdata.io](mailto://support@quiltdata.io)
+* [Slack](https://slack.quiltdata.com/)
+
+Quilt guarantees response to support issues according to the following SLAs for Quilt Business and Quilt Enterprise customers.
+
+### Quilt Business
+|  | Initial Response | Temporary Resolution |
+| ---- | ---- | ----- |
+| Priority 1 | 1 business day | 3 business days |
+| Priority 2 | 2 business days | 5 business days |
+| Priority 3 | 3 business days | N/A |
+
+### Quilt Enterprise
+|  | Initial Response | Temporary Resolution |
+| ---- | ---- | ----- |
+| Priority 1 | 4 business hours | 1 business day |
+| Priority 2 | 1 business day | 2 business days |
+| Priority 3 | 1 business days | N/A |
+
+1.	“Business Day” means Monday through Friday (PST), excluding holidays observed by Quilt Data.
```

Comment:
```suggestion
1.	*Business Day* means Monday through Friday (PST), excluding holidays observed by Quilt Data.
```
use italics instead of quotes — more consistent with rest of docs